edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/brokerConnection/IMqttBrokerConnector.cs

```diff
@@ -11,6 +11,6 @@ public interface IMqttBrokerConnector
 
         Task<bool> SendAsync(string topic, byte[] payload);
 
-        public event EventHandler OnConnected;
```

Comment:
This was removed because public is redundant in an interface, right?

Reply:
Right